softwarereview_editor.Rmd

```diff
@@ -20,32 +19,28 @@ If you're a guest editor, thanks for helping! Please contact the editor who invi
 
-- We also rotate [Editor-in-Chief responsibilities](#eicchecklist) (first-pass scope decisions and assigning editors) amongst the board about quarterly.
-
-- You do not have to keep track of other submissions, but if you do notice an issue with a package that is being handled by another editor, feel free to raise that issue directly with the other editor, or post the concern to editors-only channel on slack. Examples:
+- You only need to keep track of your own submissions, but if you do notice an issue with a package that is being handled by another editor, feel free to raise that issue directly with the other editor, or post the concern to editors-only channel on slack. Examples:
   
   - You know of an overlapping package, that hasn't been mentioned in the process yet.
-  - You see a question to which you have an expert answer that hasn't been given after a few days (e.g. you know of a blog post tackling how to add images to package docs).
-  - Concerns related to e.g. the speed of the process should be tackled by the editor-in-chief so that's who you'd turn to for such questions.
+  - You see a question to which you have an expert answer that hasn't been given after a few days (such as linking to a blog post which may answer a question).
+  - Concerns related to general review progress, including aspects such as the speed of the process, should be directed to the current Editor-in-Chief.
 
 ## Handling Editor's Checklist {#editorchecklist}
 
 ### Upon submission: {#upon-submission}
 
-- If you're the EiC or the first editor to respond, assign an editor with a comment of `@ropensci-review-bot assign @username as editor`. This will also add tag `1/editor-checks` to the issue.
-- For statistical submissions (identifiable as "Submission Type: Stats" in issue template), add the "stats" label to the issue.
-- Submission will automatically generate package check output from ropensci-review-bot which should be examined for any outstanding issues (most exceptions will need to be justified by the author in the particular context of their package.). If you want to re-run checks after any package change post a comment `@ropensci-review-bot check package`.
+- Submission will automatically generate package check output from ropensci-review-bot. The check results should be examined for any outstanding issues (most exceptions will need to be justified by the author in the particular context of their package.). Checks can be re-run after any package change with the comment `@ropensci-review-bot check package`.
+- For statistical submissions (identifiable as "Submission Type: Stats" in issue template), add the "stats" label to the issue (if not already added).
+- Check that issue template has been properly filled out. Most common oversights and omissions should be caught and noted by the bot, but a manual check always helps. Direct authors to fill any mandatory template fields that may be missing.
```

Comment:
add that the editor can also edit the issue themselves if straightforward so as to decrease the burden on authors?

Reply:
Done, thanks